install/kubernetes/helm/istio/templates/clusterrole.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.global.istioRemote }}
```

Comment:
With this flag the ClusterRole is going to be added only to `istio-remote.yaml` and not to the normal Istio.
Is this indeed the intention here?

Reply:
Yes, from my point of view, the `istio-muti` `serviceaccount` and binding `clusterrole` will only be used to add remote control plane to local control plane in multicluster deployment with VPN: https://preliminary.istio.io/docs/setup/kubernetes/multicluster/vpn/